### PR TITLE
Add docs for Schlage "Auto Lock Time"

### PR DESCRIPTION
**Pull request by mxr**
## Proposed change

Various Schlage locks support automatically locking the deadbolt after some time has elapsed. This amount of time is the "auto lock time" and it's configurable by the user. The linked PR PR adds a sensor to show the user configured value.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/123758
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Introduced a section in the Schlage integration documentation that allows users to configure the auto-lock feature, including the option to set a specific duration for automatic locking or disable it.

- **Documentation**
	- Enhanced the Schlage integration documentation with details about the "Auto Lock Time" feature, improving user understanding of its functionalities.

### CODE REVIEW

**Comment by mxr** [comment]
@emontnemery could you review this corresponding docs PR as well? Thank you

File: source/_integrations/schlage.markdown [hunk 1]
```diff
@@ -37,6 +37,12 @@ Once you have enabled the Schlage integration, you should see the following bina
 
 - **Keypad disabled** - Indicates that the keypad has been disabled, typically due to too many incorrect lock codes being attempted.
 
+## Select
+
+Once you have enabled the Schlage integration, you should see the following selects:
+
+- **Auto Lock Time** - Configure the time until the deadbolt automatically locks, or disable the auto-lock feature entirely.
```
Comment (c0ffeeca7) [inline source/_integrations/schlage.markdown:44]:
```suggestion
- **Auto Lock Time** - Configure the time until the deadbolt automatically locks, or disable the auto-lock feature entirely. For example, `0` means auto
```
```suggestion
- **Auto Lock Time** - Configure the time until the deadbolt automatically locks, or disable the auto-lock feature entirely. For example: `0`: auto-lock is disabled, `15`: it auto-locks after 15 seconds, `300`:  it auto-locks after 5 minutes.
```
Maybe an example of some possible values would be helpful here? Could you check if the suggestion is correct?

Comment (c0ffeeca7) [inline source/_integrations/schlage.markdown]:
```suggestion
- **Auto Lock Time** - Configure the time until the deadbolt automatically locks, or disable the auto-lock feature entirely. For example, `0`: auto-lock is disabled, `15`: auto-lock after 15 seconds, `300`: auto-lock after 5 minutes.
```
Would it make sense to have some examples here? Could you check if the suggestion is correct?

**Review by frenck — APPROVED** [review]
Thanks!

File: source/_integrations/schlage.markdown (hunk 1 above)
Comment (mxr) [reply source/_integrations/schlage.markdown]: This doesn't hurt, but the user will see all the options once they drill down into the `select` too. Thanks for making the change

**Pull request merged**